<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookAU</class>
 <widget class="QDialog" name="BookAU">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>283</width>
    <height>251</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>名称</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>价格</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="le_press"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>数量</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="le_cnt"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>类别1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>类别2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>类别3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_ok">
        <property name="text">
         <string>确定</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="1" colspan="2">
    <widget class="QComboBox" name="cb3">
     <item>
      <property name="text">
       <string>历史</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>言情</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人文</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>科学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>社会</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>推理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>悬疑</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>恐怖</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>轻小说</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>穿越</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>玄幻</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="cb2">
     <item>
      <property name="text">
       <string>历史</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>言情</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人文</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>科学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>社会</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>推理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>悬疑</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>恐怖</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>轻小说</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>穿越</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>玄幻</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QComboBox" name="cb1">
     <item>
      <property name="text">
       <string>历史</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>言情</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>人文</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>科学</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>社会</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>推理</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>悬疑</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>恐怖</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>轻小说</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>穿越</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>玄幻</string>
      </property>
     </item>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
